packages/compat/tests/stage1.test.ts

```diff
@@ -372,7 +372,7 @@ describe('stage1 build', function () {
         `
         treeFor(name) {
           if (name !== 'app') {
-            return this._super.treeFor(name);
+            return this._super(name);
```

Comment:
Does this replace the only test for `this._super.treeFor` (shouldn't we test both things)?

Reply:
No, I left the other test that @ef4 created: https://github.com/embroider-build/embroider/pull/572/files/0ba72bf537c5bd0fc6073751f7c6796d04ffdda2#diff-4397bded80f536069b8084a839df928911762318b820e9f6976baa1cc3f40538L353 which uses `this._super.treeFor`

Reply:
Awesome, thank you!